packages/components/src/stepper/example.md

```diff
@@ -3,8 +3,9 @@ import { Stepper } from '@woocommerce/components';
 
 const MyStepper = withState( {
     currentStep: 'first',
```

Comment:
Seems like there's a difference in spacing/tabs here that I may have introduced.  Do you mind reindenting this while you're here?